### PR TITLE
Eng 146 modifying jira plugin

### PR DESCRIPTION
**Pull request by JonRudy**

### CODE REVIEW

File: lib/logstash/inputs/jira.rb [hunk 1]
```diff
@@ -8,9 +8,16 @@
 require "rufus/scheduler"
 require "json"
 require "ostruct"
-
+require "elasticsearch"
```
Comment (ChrisSchreiber) [inline lib/logstash/inputs/jira.rb]: remove these if they are not used

Comment (ChrisSchreiber) [inline lib/logstash/inputs/jira.rb:11]: remove libraries if they are not being used

**Review by ChrisSchreiber — COMMENTED** [review]
The plugin is getting pretty complicated. I think it would be good to had commend sections to the beginning of each method that outline what the method does and what the parameters are

**Comment by JonRudy** [comment]
Updated new pr.

**Pull request closed without merging**